README.md

```diff
@@ -134,12 +134,12 @@ It will return the following blocks:
 
 The `CoreColumns` contains one or more `CoreColumn` block, and each `CoreColumn` contains a `CoreParagraph`.
 
-Given the flattened list of blocks though, how can you put it back? Well that's where you use the `nodeId` and `parentId` fields to assign temporary unique ids for each block.
+Given the flattened list of blocks though, how can you put it back? Well that's where you use the `` and `parentId` fields to assign temporary unique ids for each block.
```

Comment:
It has empty `` and `parentId`. Is `` valid? Or should that be removed? Should it read "you use the `parentId` field to assign..." instead of "you use the `` and `parentId` field(s)?